assets/js/googlesitekit/datastore/user/feature-tours.js

```diff
@@ -117,13 +195,28 @@ const baseReducer = ( state, { type, payload } ) => {
 
 const baseResolvers = {
 	*getDismissedTours() {
-		const { select } = yield Data.commonActions.getRegistry();
+		const { select } = yield getRegistry();
 		if ( ! select( STORE_NAME ).getDismissedTours() ) {
-			const { response, error } = yield fetchGetDismissedTours();
-			if ( ! error ) {
-				yield receiveGetDismissedTours( response );
+			yield fetchGetDismissedTours();
+		}
+	},
+
+	*getReadyTours( viewContext ) {
+		const registry = yield getRegistry();
+		const tours = registry.select( STORE_NAME ).getTours();
+		const readyTours = [];
+
+		for ( const tour of tours ) {
+			const tourQualifies = yield {
+				payload: { tour, viewContext },
+				type: CHECK_TOUR_REQUIREMENTS,
+			};
```

Comment:
Nit-pick, but should we maybe make this available as a (private) action too, rather than only having the action creator in here?

Reply:
I think you mean making this available as a private action creator rather than as a private action (no bound action creator exposed)? That would be a simple change to make but why would we do that unless we wanted to be able to call it elsewhere?